cprover_bindings/src/goto_program/stmt.rs

```diff
@@ -176,6 +176,7 @@ impl Stmt {
                 rhs.typ()
             );
             let assert_stmt = Stmt::assert_false(
+                "unequal-assign",
```

Comment:
I think you're probably creating too many property classes here. I think this one should be a generic "sanity check" (do we have an existing one for that?)

Reply:
yes we do! "sanity-checks" would get cleaned up by the final UI too so it makes sense to use that here.